src/modules/elixir.rs

```diff
@@ -69,22 +72,35 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     Some(module)
 }
 
-fn get_elixir_version(context: &Context) -> Option<(String, String)> {
-    let output = context.exec_cmd("elixir", &["--version"])?.stdout;
-
+fn get_elixir_version(
+    context: &Context,
+    config: &ElixirConfig,
+) -> Option<(Option<String>, String)> {
+    let output = match config.format.contains("$otp_version") {
+        true => context.exec_cmd("elixir", &["--version"])?.stdout,
+        false => context.exec_cmd("elixir", &["--short-version"])?.stdout,
```

Comment:
It would be nice to make an attempt at supporting older `elixir` versions in some manner. A possible method that has been raised is to just read the version string from the `elixir` executable script file.

Reply:
I'll need to take a closer look at this.
Don't recall that being a thing.